managed/ui/src/components/config/Storage/BackupList.js

```diff
@@ -0,0 +1,233 @@
+          currentModal={"delete" + name + "StorageConfig"}
+          visibleModal={visibleModal}
+          hideConfirmModal={hideDeleteStorageConfig}
+        >
+          Are you sure you want to delete {name} Storage Configuration?
+        </YBConfirmModal>
+      }
+
+      {/* TODO: Need to implement the show universe which is in review list */}
+      <MenuItem>
+        <i className="fa fa-eye"></i> Show Universes
+      </MenuItem>
+    </DropdownButton>
+  );
+};
+
+/**
+ * This is the header for YB Panel Item.
+ * 
+ * @param {string} currTab Active tab.
+ * @param {prop} onCreateBackup Click event.
+ */
+const header = (currTab, onCreateBackup) => (
+  <>
+    <h2 className="table-container-title pull-left">Backup List</h2>
+    <FlexContainer className="pull-right">
+      <FlexShrink>
+        <Button
+          bsClass="btn btn-orange btn-config"
+          onClick={onCreateBackup}
+        >
+          Create {currTab} Backup
+        </Button>
+      </FlexShrink>
+    </FlexContainer>
+  </>
+);
+
+const BackupList = (props) => {
+  const [ configData, setConfigData ] = useState({});  
+  const {
+    activeTab,
+    data,
+    deleteStorageConfig,
+    hideDeleteStorageConfig,
+    onCreateBackup,
+    onEditConfig,
+    showDeleteStorageConfig,
+    visibleModal
+  } = props;
+  const currTab = activeTab.toUpperCase();
+  const onEdit = onEditConfig;
+
+  return (
+    <div>
+      <YBPanelItem
+        header={header(currTab, onCreateBackup)}
+        body={
+          <>
+            <BootstrapTable
+              className="backup-list-table middle-aligned-table"
+              data={data}
+            >
+              <TableHeaderColumn
+                dataField="configUUID"
+                isKey={true}
+                hidden={true}
+              />
+              <TableHeaderColumn
+                dataField="configurationName"
+                dataFormat={getBackupConfigName}
+                columnClassName="no-border name-column"
+                className="no-border"
+              >
+                Configuration Name
+                </TableHeaderColumn>
+              <TableHeaderColumn
+                dataField="status"
+                dataFormat={getBackupStatus}
+                columnClassName="no-border name-column"
+                className="no-border"
+              >
+                Status
+                </TableHeaderColumn>
+              <TableHeaderColumn
+                dataField="backupLocation"
+                dataFormat={getBackupLocation}
+                columnClassName="no-border name-column"
+                className="no-border"
+              >
+                Backup Location
+              </TableHeaderColumn>
+              <TableHeaderColumn
+                dataField="configActions"
+                dataFormat={(cell, row) => foramtConfigActions(
+                  cell,
+                  row,
+                  {
+                    configData,
```

Comment:
it seems it would be easier to pass just `props` object or to declare `foramtConfigActions()` inside the component as it depends on many local variables

Reply:
Done.